### PR TITLE
[SCB-900]Allow number parameter to be quoted in query string

### PR DESCRIPTION
**Pull request by liubao68**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: integration-tests/it-consumer/src/main/resources/microservice.yaml [hunk 1]
```diff
@@ -16,4 +16,17 @@
 ## ---------------------------------------------------------------------------
 
 service_description:
+  environment: development
```
Comment (wujimin) [inline integration-tests/it-consumer/src/main/resources/microservice.yaml]: why need this parameter?

File: integration-tests/it-consumer/src/main/resources/microservice.yaml [hunk 2]
```diff
@@ -16,4 +16,17 @@
...
   name: it-consumer
+
```
Comment (wujimin) [inline integration-tests/it-consumer/src/main/resources/microservice.yaml]: move shared config to it-common?

File: integration-tests/it-consumer/src/main/resources/microservice.yaml (hunk 1 above)
Comment (liubao68) [reply integration-tests/it-consumer/src/main/resources/microservice.yaml]: Make local test simpler. I added some test cases, and need to delete sc's data. Add this make local development easier.

**Comment by liubao68** [comment]
Special requirement, not merge it into master.

**Pull request closed without merging**